views/about.erb

```diff
@@ -0,0 +1,39 @@
+<html>
+
+<body>
+
+<header>
+  <h1>Here's an about page</h1>
+  <nav>
+    <ul>
+      <li><a href="./index.html">Home</a></li>
+      <li><a href="./about.html">About Hailey</a></li>
+      <li><a href="./project.html">Projects</a></li>
+      <li><a href="./blog.html">Blog</a></li>
+    </ul
+  </nav>
+</header>
+
+<section>
+  <p>
+    <h3>I'm from Chicago.  It's really cold there sometimes.</h3>
+    <img src = "../chicago_is_cold.jpg"/width="40%" height="auto" alt = "Chicago is kinda cold sometimes"/>
```

Comment:
I don't think you want the extra `/` after the string which contains the photo URL string. This should just be a space.